README.md

```diff
@@ -35,15 +35,42 @@ Like all standard Jackson modules (libraries that implement Module interface), r
 ```java
 ObjectMapper mapper = new ObjectMapper();
 mapper.registerModule(new Jdk8Module());
+// Or, the more fluent version: ObjectMapper mapper = new ObjectMapper().registerModule(new Jdk8Module());
 ```
 
 after which functionality is available for all normal Jackson operations:
 you can read JSON into supported JDK8 types, as well as write values of such types as JSON, so that for example:
 
 ```java
-// TODO: real example
-Optional<String> str = ...;
-String json = mapper.writeValueAsString(str);
+class Contact {
+    @JsonProperty
```

Comment:
I personally prefer writing out the `getName()` etc methods rather than private field discovery, but that is a matter of taste I guess